es-core/src/GuiComponent.h

```diff
@@ -45,7 +45,7 @@ class GuiComponent
     void setSize(const Eigen::Vector2f& size);
     void setSize(float w, float h);
     virtual void onSizeChanged() {};
-	
+
```

Comment:
You have unrelated whitespace changes in this file. I'm happy for a separate PR to clean up some trailing whitespace if needed.